src/MissionManager/LandingComplexItem.cc

```diff
@@ -282,6 +284,10 @@ void LandingComplexItem::appendMissionItems(QList<MissionItem*>& items, QObject*
     MissionItem* item = _createDoLandStartItem(seqNum++, missionItemParent);
     items.append(item);
 
+    if (useDoChangeSpeed()->rawValue().toBool()) {
+        item = _createDoChangeSpeedItem(0, finalApproachSpeed()->rawValue().toDouble(), 0, seqNum++, missionItemParent);
```

Comment:
Should use SPEED_TYPE_AIRSPEED instead of 0. Also I think throttle should be -1. That's how the normal change speed item works: SpeedSection.cc